lib/components/narrative/narrative-itineraries.js

```diff
@@ -386,12 +388,12 @@ class NarrativeItineraries extends Component {
           showingErrors={showingErrors}
           sort={sort}
         />
-        <div
-          // FIXME: Change to a ul with li children?
-          className="list"
+        <ul
```

Comment:
I think this no longer needs to be a list (there are lists already under each mode heading).

Reply:
Unfortunately I'm realizing the issue with turning this one back into a div is that in the batch-itinerary view, the results do not show up as a list

Reply:
Which probably means if this stays as a <ul>, the sub-lists of modes in the metro-itinerary should probably also remain?